pkg/client/typed/generated/extensions/unversioned/extensions_client.go

```diff
@@ -46,10 +45,6 @@ func (c *ExtensionsClient) Deployments(namespace string) DeploymentInterface {
 	return newDeployments(c, namespace)
 }
 
-func (c *ExtensionsClient) HorizontalPodAutoscalers(namespace string) HorizontalPodAutoscalerInterface {
```

Comment:
Not changing as explained in https://github.com/kubernetes/kubernetes/pull/20501#issuecomment-180260757